src/styles/stars-block.css

```diff
@@ -0,0 +1,40 @@
+.stars-block {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.stars {
```

Comment:
it's not clear why did you create 2 separate BEM blocks for starts?
Will be better to leave only one BEM block, and make other blocks an elements